css/style.scss

```diff
@@ -621,6 +621,20 @@ input.input-inline {
 		font-size: 8pt !important;
 		display: none;
 		align-self: flex-end;
+		min-width: 120px;
+		text-align: center;
+	}
+	.unsaved-indicator {
```

Comment:
I would prefer having a new common class for both indicators because they share 80% of the declarations.